packages/asc-ui-beta/src/utils/betaUtils.ts

```diff
@@ -59,10 +59,7 @@ export const fluidTypoStyle = css`
 `
 
 export interface TypoStyle {
-  gutterBottom?: {
-    small: number
-    large: number
-  }
+  gutterBottom?: any
```

Comment:
Any particular reason for the prop `butterBottom` not to have a type anymore?

Reply:
Yeah, the previous typing didn't work for both an object and a number :) I couldn't get the typing to work correctly, so I removed it for now. It's on my 'fix later' list